Documentation/CodingGuidelines/CglTypoScript/Index.rst

```diff
@@ -9,32 +9,28 @@ TypoScript coding guidelines
 Directory and file names
 ========================
 
-* The file extension **must** be :file:`.typoscript`.
+..  versionchanged:: 12.0
+    Support for other TypoScript file extensions, such as :file:`.txt` or
+    :file:`.ts`, have been removed in TYPO3 v12.
```

Comment:
I think it should be "has" not "have" because "support" is removed, not a plural "file extensions"?

```suggestion
    :file:`.ts`, has been removed in TYPO3 v12.
```

Reply:
Oh, yes, focused on the "extensions". Sorry.